specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json

```diff
@@ -0,0 +1,431 @@
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Authorization/dataPolicyManifests/{policyMode}": {
+      "get": {
+        "tags": [
+          "DataPolicyManifests"
+        ],
+        "operationId": "DataPolicyManifests_GetByPolicyMode",
+        "summary": "Retrieves a data policy manifest.",
+        "description": "This operation retrieves the data policy manifest with the given policy mode.",
+        "x-ms-examples": {
+          "Retrieve a data policy manifest by policy mode": {
+            "$ref": "./examples/getDataPolicyManifest.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "policyMode",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The policy mode of the data policy manifest to get."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about a data policy manifest.",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyManifest"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Authorization/dataPolicyManifests": {
+      "get": {
+        "tags": [
+          "DataPolicyManifests"
+        ],
+        "operationId": "DataPolicyManifests_List",
+        "summary": "Retrieves data policy manifests",
+        "description": "This operation retrieves a list of all the data policy manifests that match the optional given $filter. Valid values for $filter are: '$filter=namespace eq '{0}''. If $filter is not provided, the unfiltered list includes all data policy manifests for data resource types. If $filter=namespace is provided, the returned list only includes all data policy manifests that have a namespace matching the provided value.",
+        "x-ms-examples": {
+          "List data policy manifests": {
+            "$ref": "./examples/listDataPolicyManifests.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/DataPolicyManifestsFilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of data policy manifests.",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyManifestListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
```

Comment:
we don't actually support paging on this api - we always return a null nextLink

Reply:
It's ok. That looks like you only return 1 page. If you ever need to add paging, you can just change your service and there are no client breaking changes. See [https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls) for details.

---
In reply to: [512959200](https://github.com/Azure/azure-rest-api-specs/pull/11391#discussion_r512959200) [](ancestors = 512959200)